core/templates/dev/head/library/Library.js

```diff
@@ -108,6 +108,29 @@ oppia.controller('Library', [
 
     var isAnyCarouselCurrentlyScrolling = false;
 
+    var highlightText = null;
+
+    $scope.highlight = function(index) {
```

Comment:
Let's call this highlightGroup(groupIndex) (so we know what the index refers to, etc.)